src/Generators/ModelGenerator.php

```diff
@@ -235,4 +282,441 @@ public function rollback()
             $this->commandData->commandComment('Model file deleted: '.$this->fileName);
         }
     }
+
+    private function getEloquentRules()
```

Comment:
Thought about creating a stand alone generateEloquent class. Perhaps in next rev.